jello/lib.py

```diff
@@ -396,7 +396,7 @@ def read_file(file_path):
     with open(file_path, 'r') as f:
         return f.read()
 
-def pyquery(data, query):
+def pyquery(data, query, add_to_scope={}):
```

Comment:
Since `add_to_scope` is mutable it would be preferable to set it to `None` and then change to a dictionary within the function. Probably doesn't matter in this case since the function will only be called once, but probably good from a hygiene standpoint.

Reply:
Yep that works great. The other way would be 
```
if add_to_scope is None:
    add_to_scope = {}
```

Reply:
when you add a line with "if" I'd say it is better to not modify the scope at all in the `None` case.

I updated the code as suggested.